### DISCUSSION TITLE
what keyboard combination is free transform?

### DISCUSSION THREAD

**Discussion by Thetaco** [obsproject/obs-studio#4610] [Development Questions] [answered]
**what keyboard combination is free transform?
**trying to figure out OBS but more experience I have in what is called adobe photoshop

**Comment by RytoEX** [comment, marked as answer]
Hold the "Control" key while using the mouse to transform a source.  In the future, our Discord or forums is probably a more appropriate place to ask for application usage help.
https://obsproject.com/help

**Reply by Thetaco** [reply]
thank you so much for answering my question. I thought control was the answer, thank you I did not mean to be rude, your open source broadcasting is so great, I just learned on a newtek-tricaster.

I apologize for posting here in haste. I had played around with OBS for some time now and realized that indeed the control was similar. I just hope people with experience in 'adobe photoshop' understand the mouse feel of "control+t", i dont know much of anything about programming so I guess it is what you all call, 'resource intensive?'. I apologize!!!
